releasenotes/v0.10.0-beta.0.md

```diff
@@ -0,0 +1,145 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v0.9.0
+
+:rotating_light: This is a BETA RELEASE. Use it only for testing purposes.
+If you find any bugs, file an [issue](https://github.com/metal3-io/baremetal-operator/issues/new/).
+
+## :warning: Breaking Changes
+
+- Bump Kubernetes api, controller-runtime and decouple webhooks from api (#2390)
+
+<details>
+<summary>More details about the release</summary>
+
+## :sparkles: New Features
+
+- Add support for Disablepoweroff (#2229)
+- add support for locally built ironic in e2e test (#2167)
```

Comment:
Nit: I know it was marked as feature, but I think this would fit better as :seedling: since it is just about the tests.